components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -143,7 +143,7 @@ - (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
   XCTAssertTrue(self.manager.internalManager.overlayView.accessibilityViewIsModal);
 }
 
-- (void)testSnackbarAccessibiltyViewIsModalShouldBeNoWithNoActions {
+- (void)testSnackbarAccessibiltyViewIsModalShouldBeYesWithNoActions {
```

Comment:
This looks like a breaking change for clients. Should we announce this internally?

Reply:
in the test they are explicitly setting shouldEnableAccessibilityViewIsModal = YES. I think the name of the test is confusing and makes it sound like the default is YES.

Reply:
`testSnackbarAccessibiltyViewIsModalShouldBeYesWithNoActions` has been renamed to `testSnackbarAccessibiltyViewIsModalAppliedWithNoActions` Addressed in ca3870742a2cab331e3107faa76fb8868801ac82